### PR TITLE
#WCD-8694|show electron when editor fails to load

### PR DESCRIPTION
**Pull request by tombenezra**

### CODE REVIEW

**Comment by dor-itzhaki** [comment]
what's stopping us from merging this ?

**Comment by tombenezra** [comment]
if you could approve the pr, I think it's mergable

File: packages/corvid-cli/src/apps/pull.js [hunk 1]
```diff
@@ -50,6 +50,16 @@ const pullApp = ({ useSsl = true, override = false, move = false } = {}) => ({
         isHeadless
       );
 
+      win.webContents.on("did-finish-load", async () => {
+        const editorLoadingFailed = await win.webContents.executeJavaScript(
+          "window.editorModel === undefined"
```
Comment (tomenden) [inline packages/corvid-cli/src/apps/pull.js]: Please check with @EtaiG / Editor folks that this is a legit way to verify if the editor is loaded (maybe @inballa knows?)

File: packages/corvid-e2e-tests/package.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "test": "run-script-os",
     "test:windows": ":",
-    "test:default": "jest --maxWorkers=1 --detectOpenHandles"
+    "test:default": "jest --maxWorker=1 --runInBand --detectOpenHandles"
```
Comment (tomenden) [inline packages/corvid-e2e-tests/package.json]: why the `runInBand`?

File: packages/corvid-e2e-tests/setupJest.js [hunk 3]
```diff
@@ -1 +1 @@
-jest.setTimeout(200000);
+jest.setTimeout(300000);
```
Comment (tomenden) [inline packages/corvid-e2e-tests/setupJest.js]: Why this change?

File: packages/corvid-e2e-tests/test/clone.e2e.js [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+const localEditorDriverCreator = require("./drivers/localEditorDriver");
+
+const testSites = [
+  {
+    editorUrl:
+      "https://editor.wix.com/html/editor/web/renderer/edit/cfbebf57-9622-4c13-9b67-2e6e55b2c265?metaSiteId=de7cb190-770a-46a5-8ef8-914bb8cf727c&editorSessionId=e38c6558-de49-548a-8860-f128da278c7c",
+    description: "not permitted",
+    errorMessage: "(Error 403)"
+  }
+  // {
```
Comment (tomenden) [inline packages/corvid-e2e-tests/test/clone.e2e.js]: Is the comment part left here on purpose?

File: packages/corvid-e2e-tests/test/clone.e2e.js [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+        const loginDebugPort = await findFreePort();
+
+        const loginCommand = cliDriver.login({
+          remoteDebuggingPort: loginDebugPort
+        });
+        const loginEditorDriver = await connectToLocalEditor(loginDebugPort);
+        await loginEditorDriver.waitForLoginForm();
+
+        await loginEditorDriver.login({
+          username: "corvidtest@gmail.com",
```
Comment (tomenden) [inline packages/corvid-e2e-tests/test/clone.e2e.js]: We should think about this, not sure it's a good idea to publicize it, though I'm not sure it's that terrible ATM either.. @dor-itzhaki WDYT?
